public/data/rp2022.json

```diff
@@ -174,24 +174,32 @@
             "questions": [
```

Comment:
Not actually sure this file is used?  I see where it's loaded, but didn't bother tracing it further after finding the blobs in About/index